plugin/src/ast_helpers.rs

```diff
@@ -196,7 +197,10 @@ fn string_node_equals(node: Node, value: &str, context: &QueryMatchContext) -> b
 }
 
 pub fn is_class_member_static(node: Node, context: &QueryMatchContext) -> bool {
-    assert_kind!(node, MethodDefinition | FieldDefinition);
+    assert_kind!(
+        node,
+        MethodDefinition | FieldDefinition | "public_field_definition" // I guess Typescript uses this instead of FieldDefinition?
```

Comment:
As of now I'm just doing the "easiest" thing and adding Typescript-specific stuff to some helpers here 

That at least sounds more appealing than copying + pasting these into the Typescript plugin with slight modifications 

Though maybe raises some broader questions of "Typescript-awareness" in this plugin. Eg if there are rules that typescript-eslint has to copy + paste and slightly modify but that could "nicely" contain those modifications in the base plugin then might be nicer to do that?